src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -18,7 +32,10 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
         private _orderActionCreator: OrderActionCreator,
         private _paymentActionCreator: PaymentActionCreator,
         private _paypalCommercePaymentProcessor: PaypalCommercePaymentProcessor,
-        private _paypalCommerceFundingKeyResolver: PaypalCommerceFundingKeyResolver
+        private _paypalCommerceFundingKeyResolver: PaypalCommerceFundingKeyResolver,
+        private _paypalCommerceRequestSender: PaypalCommerceRequestSender,
+        private _pollingInterval?: number,
+        private _pollingTimer: number = 0
```

Comment:
🍹 ts should handle correct type according to default value
```suggestion
        private _pollingTimer = 0
```

Reply:
fixed